packages/@okta/vuepress-site/docs/guides/session-cookie/main/index.md

```diff
@@ -24,7 +24,9 @@ Okta uses an HTTP session cookie to provide access to your Okta org and apps acr
 
 > **Note:** Some browsers have begun blocking third-party cookies by default, which disrupts Okta functionality in certain flows. See [FAQ: How Blocking Third Party Cookies Can Potentially Impact Your Okta Environment](https://support.okta.com/help/s/article/FAQ-How-Blocking-Third-Party-Cookies-Can-Potentially-Impact-Your-Okta-Environment).
 
-Okta sessions are created and managed with the [Session API](/docs/reference/api/sessions/).
+Okta sessions are created and managed with the [Session API](/docs/reference/api/sessions/). A session token is sent as part of a request, contained in a `sessionToken` parameter. If the request is successful, the session cookie is set by a `Set-Cookie` header in the response.
+
+> **Important**: By default, Okta Classic orgs ignore the `sessionToken` in a request if there is already a session cookie set in the browser. It is possible to flip this behavior in a specific org by setting an internal flag; talk to your Okta rep to find out more.
```

Comment:
```suggestion
> **Important**: By default, Okta Classic orgs ignore the `sessionToken` in a request if there is already a session cookie set in the browser. If required, you can change this default behavior for a specific org. Contact your Okta rep for more information.
```

Reply:
Suggestion for the last sentence. There was some talk of standardizing on a "contact your Okta rep" terminology, but not sure we decided on it yet. Approving if this is something we can work out later.